.eslintrc.json

```diff
@@ -1,15 +1,11 @@
 {
     "extends": [
         "eslint:recommended",
-        "plugin:@typescript-eslint/recommended",
-        "prettier",
-        "prettier/@typescript-eslint"
+        "plugin:@typescript-eslint/recommended"
     ],
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["@typescript-eslint"],
```

Comment:
The above two lines are unneeded because of `plugin:@typescript-eslint/recommended` configures them.

https://github.com/typescript-eslint/typescript-eslint/blob/7b701a3091eca0eef5a1e3a7cbcadbabdc54104d/packages/eslint-plugin/src/configs/base.ts#L5-L9